wordpress-mobile/WordPressKit-iOS
# Remove AFNetworking from WPStatsServiceRemote

Refs #4 

This PR removes the AFNetworking code being used on `WPStatsServiceRemote` class and replaces it with vanilla NSURLSession.

How to test:
 - Check that unit tests are passing.
 - Check in the main app if stats are working correctly using the branch of this [PR on the main repo](https://github.com/wordpress-mobile/WordPress-iOS/pull/9523).
